src/site/content/en/blog/largest-contentful-paint/index.md

```diff
@@ -0,0 +1,264 @@
+of the viewport, or if any of the element is clipped or has non-visible
+[overflow](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow), those
+portions do not count toward the element's size.
+
+For image elements, only the visible size of the rendered image is considered.
+The original size of the image is not factored into the element size
+calculation, nor is any margin, padding, or border that is applied to the image
+via CSS.
+
+For text elements, only the size of its text nodes is considered (the smallest
+rectangle that encompases all text nodes). As with images, the element's
+margin, padding, and border applied via CSS are not considered part of its
+contentful size.
+
+{% Aside %}
+  Determining which text nodes belong to which elements can sometimes
+  be tricky, especially for elements whose children includes inline elements and
+  plain text nodes but also block-level elements. The key point is that every
+  text node belongs to (and only to) its closest block-level ancestor element.
+  In spec terms: each text nodes belong to the element that generates its
+  containing block.
+{% endAside %}
+
+### When is largest contentful paint reported?
+
+Web pages often load in stages, and as a result, it's possible that the largest
+element on the page might change.
+
+To handle this potential for change, the browser dispatches a
+[`PerformanceEntry`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry)
+of type `largest-contentful-paint` identifying the largest contentful element
+as soon as the browser has painted the first frame. But then, after rendering
+subsequent frames, it will dispatch another
+[`PerformanceEntry`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry)
+any time the largest contentful element changes.
+
+It's important to note that an element can only be considered the largest
+contentful element once it has rendered and is visible to the user. Images that
+have not yet loaded are not considered "rendered". Neither are text nodes using
+web fonts during the [font block
+period](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display#The_font_display_timeline).
+In such cases a smaller element may be reported as the largest contentful
+element, but as soon as the larger element finishes rendering, it'll be
+reported via another  `PerformanceEntry` object.
+
+In addition to late-loading images and fonts, a page may add new elements to
+the DOM as new content becomes available. If any of these new elements is
+larger than the previous largest contentful element, a new `PerformanceEntry`
+will also reported.
+
+The browser will stop reporting new entries as soon as the user interacts with
+the page (via a tap, scroll, or keypress), as user interaction often changes
+what's visible to the user (which is especially true with scrolling).
+
+For analysis purposes, only the most recently dispatched `PerformanceEntry`
+object should be reported to your analytics service.
```

Comment:
This is something the page developer is doing in code, right? This also needs to be active voice. Both problems have the same solution. "For analysis purposes, you should only report..."

Reply:
Done.